base/ltshipout.dtx

```diff
@@ -31,16 +31,16 @@
 %
 %%% From File: ltshipout.dtx
 %
-%    \begin{macrocode}
-\providecommand\ltshipoutversion{v1.0n}
-\providecommand\ltshipoutdate{2022/11/08}
-%    \end{macrocode}
-%
 %<*driver>
-
+% \fi
+\ProvidesFile{ltshipout.dtx}
+             [2024/02/11 v1.0n LaTeX Kernel (Shipout)]
+% \iffalse
+%
 \documentclass{l3doc}
+\GetFileInfo{ltshipout.dtx}
 
-% bug fix fo l3doc.cls
+% bug fix for l3doc.cls
 \ExplSyntaxOn
 \cs_set_protected:Npn \__codedoc_macro_typeset_one:nN #1#2
   {
```

Comment:
is that bug fix still relevant? or fixed?

Reply:
Good questions: I'll check that (independent of PR)

Reply:
The missing-`\MacroFont` bug has been fixed in `l3doc.dtx` in latex3/latex3@34338536 committed on 2020-08-21, and then simplified in https://github.com/latex3/latex3/commit/a67bf5341092ba962c88681c1618d768f8f360c4.

Thus the redefinition of `\__codedoc_macro_typeset_one:nN` in `ltshipout.dtx` can be safely removed.